PyInstaller/utils/hooks/qt.py

```diff
@@ -219,7 +219,7 @@ def get_qmake_path(version=''):
         return os.path.join(os.environ['QT4DIR'], 'bin', 'qmake')
 
     # try the default $PATH
-    dirs = ['']
+    dirs = ['/']
```

Comment:
Why did you change this? I can see `dirs - ['', '/']`, but not simply removing `''`.

Reply:
Good point!

Reply:
Corrections done.